sympy/functions/combinatorial/numbers.py

```diff
@@ -1984,3 +1984,123 @@ def nT(n, k=None):
     for discard in m.enum_range(n[_M], k-1, k):
         tot += 1
     return tot
+
+#-----------------------------------------------------------------------------#
+#                                                                             #
+#                          Motzkin numbers                                    #
+#                                                                             #
+#-----------------------------------------------------------------------------#
+class motzkin(Function):
+    """
+    In mathematics, the nth Motzkin number is the number
+    of different ways of drawing non-intersecting chords
+    between n points on a circle (not necessarily touching
+    every point by a chord). The Motzkin numbers are named
+    after Theodore Motzkin and have diverse applications
+    in geometry, combinatorics and number theory.
+
+    Motzkin numbers satisfy this recurrence relation
+    M(n) = ( (2n+1) / (n+2) )*M(n-1)  +  ( (3n-3) / (n+2) )*M(n-2)
+
+
+    Examples
+    ========
+
+    >>> from sympy import motzkin
+    >>> motzkin.find_first_n_motzkin(10)
+        [1, 1, 2, 4, 9, 21, 51, 127, 323, 835]
+
+    >>> motzkin.find_motzkin_numbers_in_range(100,1000)
+        [127, 323, 835]
+
+    >>> motzkin.is_motzkin(100)
+        False
+
+    >>> motzkin.is_motzkin(51)
+        True
+
+    >>> motzkin.eval_motzkin(10)
```

Comment:
like `prime(3)->5` how about `motzkin.motzkin(10)->835`?

Reply:
okay i will change it right away